specification/iothub/resource-manager/Microsoft.Devices/stable/2020-03-01/iothub.json

```diff
@@ -2281,6 +2281,10 @@
             "$ref": "#/definitions/IpFilterRule"
           }
         },
+        "minTlsVersion": {
```

Comment:
>minTlsVersion [](start = 9, length = 13)

You are adding a new property to an existing API version.

What happens when the user is using a client that was built before the property was added? Is there a possibility of data loss?

Reply:
This property was already being returned our service but was just not added into the swagger (due to a mistake?)

We're facing a situation in production where clients (including our own Azure portal extension) are getting errors because the property is returned in the API response but not built into the client (because it's missing in the swagger)

I don't believe there's possibility for data loss.

Reply:
Perfect. Thanks for confirming.

Reply:
Yes, this property was already being accepted and returned by the service but was not included in the swagger spec. This PR basically addresses that gap.